main.py

```diff
@@ -1,14 +1,43 @@
 import os
 import sys
+
+from kubernetes import config
+import kubernetes.client
+import yaml
+import logging
 from config import parse_config
 from config import validate
-from kubernetes import config
+from config import validate_user_rule
 
-import logging
 logging.basicConfig(level=os.environ.get('LOG_LEVEL', logging.INFO))
 log = logging.getLogger(__name__)
 
 
+def read_config_files(configuration):
+    """
+    Read configuration files from configmaps in all available namespaces
+    :param configuration: Kubernetes client configuration
+    :return: list of user elastalert rules
+    """
+    api_instance = kubernetes.client.CoreV1Api(kubernetes.client.ApiClient(configuration))
+    configmap_list = api_instance.list_config_map_for_all_namespaces()
```

Comment:
Administrator will restrict for reading all configmaps from any namespaces. So we should write exact query.

Reply:
Changed
```suggestion
    configmap_list = api_instance.list_config_map_for_all_namespaces()
```